components/automate-cli/pkg/verifyserver/server/api/v1/nfsmount_test.go

```diff
@@ -17,7 +17,13 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func SetupNFSMountHandler(nm nfsmountservice.INFSService) (*fiber.App, error) {
+var (
+	nfsErrorBodyParser = "{\"status\":\"FAILED\",\"result\":null,\"error\":{\"code\":400,\"message\":\"Invalid Body Request\"}}"
+	errorMountLocEmpty = "{\"status\":\"FAILED\",\"result\":null,\"error\":{\"code\":400,\"message\":\"Mount Location cannot be empty\"}}"
+	successMsg         = "{\"status\":\"SUCCESS\",\"result\":{\"address\":\"\",\"mount_location\":\"/data\",\"nfs\":\"\",\"storage_capacity\":\"\",\"Available_free_space\":\"\"}}"
```

Comment:
Available_free_space
```suggestion
	successMsg         = "{\"status\":\"SUCCESS\",\"result\":{\"address\":\"\",\"mount_location\":\"/data\",\"nfs\":\"\",\"storage_capacity\":\"\",\"available_free_space\":\"\"}}"
```